jcstress-core/src/main/java/org/openjdk/jcstress/infra/processors/JCStressTestProcessor.java

```diff
@@ -676,8 +676,9 @@ private boolean allFieldsAreDefault(TypeElement el) {
                 if (t.getInitializer() != null) return false;
             }
             if (member.getKind() == Tree.Kind.BLOCK) {
+                BlockTree b = (BlockTree)member;
```

Comment:
Sorry, a space here `(BlockTree) member`.

Reply:
No problem. I'm all for code sanity :)